test/recipes/30-test_evp_data/evppkey.txt

```diff
@@ -635,7 +635,7 @@ Result = KEYOP_INIT_ERROR
 Reason = invalid salt length
 
 
-Title = EC tests
+Title = EC DSA tests
```

Comment:
Is it intended to have a space between EC and DSA?

Reply:
removed